web-performance/README.md

```diff
@@ -0,0 +1,28 @@
+Web Performance
+===============
+
+Web performance refers to the speed in which web pages are downloaded and
+displayed on the user's web browser. Web performance optimization (WPO) or
+website optimization is the field of knowledge about increasing web performance.
+
+
+Resources
+---------
+
+* [Demystifying Speed Tooling (Google I/O ’19)](https://www.youtube.com/watch?v=mLjxXPHuIJo)
+* [Fast load times](https://web.dev/fast)
+* [Your first performance budget with Lighthouse](https://bitsofco.de/your-first-performance-budget-with-lighthouse/)
```

Comment:
Thoughts on using reference-style links, which can help to keep the raw content more parse-able by placing long URLs elsewhere in the document:

```
* [Demystifying Speed Tooling (Google I/O ’19)][google-io-video]
* [Fast load times][web-dev-fast]
* [Your first performance budget with Lighthouse][lighthouse-budgets]

[google-io-video]: https://www.youtube.com/watch?v=mLjxXPHuIJo
[web-dev-fast]: https://web.dev/fast
[lighthouse-budgets]: https://bitsofco.de/your-first-performance-budget-with-lighthouse/
```

Reply:
I love this! Didn't know this was an option either. Thanks for sharing 😎